addon/mixins/flexberry-ddau-slider-actions-handler.js

```diff
@@ -56,6 +58,9 @@ export default Ember.Mixin.create({
         mutablePropertyPathType === 'string');
 
       setRecord(this, mutablePropertyPath, e.newValue);
+
+      // Save changes to local storage.
+      this.mutateStorage('opacity', mutablePropertyPath, e.newValue);
```

Comment:
we can get bag here if app programmer extend class with this mixin and dont extend with local-storage-binding.

Reply:
So, what do you propose? Extending mixin with mixin is bad idea. This way, edit-map controller uses local-storage-binding, so applied programmer do not have to do it.